docs/insomnia/requests.md

```diff
@@ -0,0 +1,97 @@
+1. In the **New Request** modal:
+   1. Enter a request name.
+   1. Select a method from the dropdown.
+   1. Click **Create**.
+1. Enter the API URL in the request URL input.
+1. To add authentication, go to the **Auth** tab. Use the **Auth** dropdown to select your authentication method.
+1. When you have entered your request URL and authentication, click **Send**.
+
+See [Request Options](#request-options) to customize your request.
+
+## Send a gRPC request
+
+Send a gRPC method request in a Document or Collection. The following instructions
+assume you're starting with an empty Document or Collection.
+
+1. In a Document, select the **Debug** tab, then **New Request**. In a Collection, select **New Request**.
+1. In the **New Request** modal:
+   1. Enter a request name.
+   1. Select **gRPC** from the dropdown.
+   1. Click **Create**.
+1. You'll be taken to a **Select Proto File** modal.
+1. Click **Add Directory** or **Add Proto File**.
+1. Add your directory or upload your Proto File. Insomnia will automatically detect the streaming types and modify the first tab as such.
+1. Enter your API URL and click **Start**.
+
+See [gRPC Request Options](#grpc-request-options) to customize your request.
+
+## gRPC Request Options
+
+Modify any gRPC request with the following options.
+
+{:.table .table-striped}
+Option | Description
+------ | ------------
+*Streaming Type* <small>Example: `Unary`</small> | The body of the request. The tab displays the streaming type, auto-detected from the Proto File or directory.<br><br>Streaming types are: Unary, Server Streaming, Client Streaming, and Bidirectional Streaming.
+Headers | Add a header to your request. For example, `Content-Type: application/grpc`.
+
+## Send a GraphQL request
+
+Send a GraphQL method request in a Document or Collection. The following instructions
+assume you're starting with an empty Document or Collection.
+
+1. In a Document, select the **Debug** tab, then **New Request**. In a Collection, select **New Request**.
+1. In the **New Request** modal:
+   1. Enter a request name.
+   1. Select **POST** from the dropdown.
+   1. Another dropdown will appear to the right. Open it and select **GraphQL Query**.
+   1. Click **Create**.
+1. Enter the API URL in the request URL input.
+1. In the GraphQL body tab, add you GraphQL schema.
+1. To add authentication, go to the **Auth** tab. Use the **Auth** dropdown to select your authentication method.
+1. When you have entered your request URL and authentication, click **Send**.
+
+See [Request Options](#request-options) to customize your request.
+
+## Request Options
+
+Modify any HTTP method or GraphQL request with the following options.
+
+{:.table .table-striped}
+Option | Description
+------ | ------------
+Body | Select a body type from the Body dropdown menu on the tab. Add content that you want to send in the body of the request.
```

Comment:
This part of the table is a bit confusing. Why do we write 2 sentences for the descriptions of the `Body` row and the `Auth` row?

Reply:
I always lean towards being explicit, and sometimes that means it takes 2 sentences to explain something. Let me know what you think is confusing so I can clarify or edit what's here!

Reply:
@falondarville ignore my last comment, I re-read those, and I now see what was the intent. My initial confusion was due to the sentences seeming very similar, and I thought maybe the repetition was not intended, but I can see now what was trying to be achieved, and that they are not the same.